obspy/clients/nrl/client.py

```diff
@@ -0,0 +1,311 @@
+        cp = self._get_cp_from_ini(path)
+        for section in cp.sections():
+            options = sorted(cp.options(section))
+            if section.lower() == 'main':
+                if options not in (['question'],
+                                   ['detail', 'question']):  # pragma: no cover
+                    msg = "Unexpected structure of NRL file '{}'".format(path)
+                    raise NotImplementedError(msg)
+                nrl_dict._question = self._clean_str(cp.get(section,
+                                                            'question'))
+                continue
+            else:
+                if options == ['path']:
+                    nrl_dict[section] = NRLPath(self._choose(section, path))
+                    continue
+                # sometimes the description field is named 'description', but
+                # sometimes also 'descr'
+                elif options in (['description', 'resp'], ['descr', 'resp'],
+                                 ['resp']):
+                    if 'descr' in options:
+                        descr = cp.get(section, 'descr')
+                    elif 'description' in options:
+                        descr = cp.get(section, 'description')
+                    else:
+                        descr = '<no description>'
+                    descr = self._clean_str(descr)
+                    resp_path = self._choose(section, path)
+                    nrl_dict[section] = (descr, resp_path)
+                    continue
+                else:  # pragma: no cover
+                    msg = "Unexpected structure of NRL file '{}'".format(path)
+                    raise NotImplementedError(msg)
+        return nrl_dict
+
+    def _clean_str(self, string):
+        return string.strip('\'"')
+
+    def get_datalogger_resp(self, datalogger_keys):
+        """
+        Get the RESP string of a datalogger by keys.
+
+        :type datalogger_keys: list of str
+        :rtype: str
+        """
+        datalogger = self.dataloggers
+        while datalogger_keys:
+            datalogger = datalogger[datalogger_keys.pop(0)]
+        return self._read_resp(datalogger[1])
+
+    def get_sensor_resp(self, sensor_keys):
+        """
+        Get the RESP string of a sensor by keys.
+
+        :type sensor_keys: list of str
+        :rtype: str
+        """
+        sensor = self.sensors
+        while sensor_keys:
+            sensor = sensor[sensor_keys.pop(0)]
+        return self._read_resp(sensor[1])
+
+    def get_parser(self, datalogger_keys, sensor_keys):
+        """
+        Get  io.xseed.Parser for RESP
+
+        :type datalogger_keys: list of str
+        :type sensor_keys: list of str
+        :rtype: :class:`~obspy.io.xseed.Parser`
+        """
+        dl_parser = Parser(self.get_datalogger_resp(datalogger_keys))
+        sensor_parser = Parser(self.get_sensor_resp(sensor_keys))
+        return Parser.combine_sensor_dl_resps(sensor_parser, dl_parser)
```

Comment:
Might be good to change this method name to something more verbose like `Parser.combine_sensor_and_datalogger_resp()`